Name: Disable Landscape Gesture Scroll 
Version: 1.3.5-1
Author: Fritos1406
Description: Disable the scroll feature using the gesture area in landscape

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
@@ -90,7 +90,7 @@
 		this._updateOrientation();
 
 		// Allow override of 'back' gesture in landscape mode.
-		this.controller.useLandscapePageUpDown(true);
+		this.controller.useLandscapePageUpDown(false);
 
 		// Create the browser server status control.
 		this._browserServerStatus = new BrowserServerStatus(this.controller);
--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/startpage-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/startpage-assistant.js
@@ -23,7 +23,7 @@
 	this._updateOrientation();
 
 	// Allow override of 'back' gesture in landscape mode.
-	this.controller.useLandscapePageUpDown(true);
+	this.controller.useLandscapePageUpDown(false);
 
 	// Listen for scene scope keyboard events.
 	this.controller.listen(this.controller.sceneElement, Mojo.Event.keydown, this._onKeyDownEventHandler);
 
